cmake/SetupSpec.cmake

```diff
@@ -3,14 +3,14 @@
 
 find_package(SpEC)
 
-if (NOT SpEC_FOUND)
-  return()
-endif()
-
 # Make SpEC scripts available in Python. These can be used until we have ported
 # them to SpECTRE.
 set(PYTHONPATH "${SPEC_ROOT}/Support/Python:${PYTHONPATH}")
 
+if (NOT SpEC_FOUND)
```

Comment:
You have to wrap the line above in `if (SPEC_ROOT)` though

Reply:
Oops. Done